invader_payment_adyen/models/payment_transaction.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 ACSONE SA/NV (http://acsone.eu).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class PaymentTransaction(models.Model):
+    _inherit = "payment.transaction"
+
+    adyen_payment_data = fields.Char(groups="base.group_user")
```

Comment:
I do not catch the use of "groups" attribute

Reply:
I replicated Odoo standard fields attributes